Formula/libcouchbase.rb

```diff
@@ -29,7 +29,7 @@ def install
     args << "-DLCB_NO_TESTS=1"
 
     ["libev", "libevent", "libuv"].each do |pname|
-      args << "-DLCB_BUILD_#{pname.upcase}=" + (build.with?("#{pname}") ? "ON" : "OFF")
+      args << "-DLCB_BUILD_#{pname.upcase}=" + (build.with?(pname.to_s) ? "ON" : "OFF")
```

Comment:
In this case I `pname` is already a string so you can just use `pname`. Also, `package_name` may be a better choice.